src/vcsparser.core/git/GitWorkItemConverter.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using vcsparser.core.bugdatabase;
+
+namespace vcsparser.core.git
+{
+    public class GitWorkItemConverter : IWorkItemConverter
+    {
+        private readonly ICommandLineParser commandLineParser;
+        private readonly IProcessWrapper processWrapper;
+        private readonly IGitLogParser gitLogParser;
+
+        private readonly string gitShowCommand;
+
+        public GitWorkItemConverter(ICommandLineParser commandLineParser, IProcessWrapper processWrapper, IGitLogParser gitLogParser)
+        {
+            this.commandLineParser = commandLineParser;
+            this.processWrapper = processWrapper;
+            this.gitLogParser = gitLogParser;
+
+            this.gitShowCommand = "git show --pretty=fuller --date=iso --numstat {0}";
+        }
+
+        private IChangeset Convert(WorkItem item)
+        {
+            // TODO Check if changeset is in hash format

```

Comment:
Is this intentionally here?

Reply:
File Removed in 4d01be55a554ac9100e37105327287170fdfd4eb